mondey_backend/src/mondey_backend/routers/admin_routers/calendarevents.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from fastapi import APIRouter
+
+from ...dependencies import SessionDep
+from ...models.calendarevents import CalendarEvent
+from ...models.calendarevents import CalendarEventCreate
+from ...models.calendarevents import CalendarEventRead
+from ...models.calendarevents import CalendarEventUpdate
+from ..utils import add
+from ..utils import get
+
+
+def create_router() -> APIRouter:
+    router = APIRouter()
+
+    @router.post("/calendarevents/", response_model=CalendarEventRead)
+    def create_event(session: SessionDep, event: CalendarEventCreate):
+        db_event = CalendarEvent.model_validate(event)
+        add(session, db_event)
+        return db_event
```

Comment:
probably doesn't matter but to make sure the returned db_event has the correct id I think we need a `session.refresh(db_event)` between adding it and returning it